nicos_ess/devices/epics/extensions.py

```diff
@@ -66,6 +66,15 @@ def _readRaw(self, maxage=0):
     def _startRaw(self, target):
         EpicsDigitalMoveableEss.doStart(self, target)
 
+    def doStart(self, value):
+        if value in self.mapping.keys():
```

Comment:
```suggestion
        if value in self.mapping:
```
Does that work? I think the mapping is either a dict or inherits from dict.